docs/Manual/Deployment/Kubernetes/DeploymentResource.md

```diff
@@ -413,6 +413,10 @@ for each server of this group.
 This setting is not available for group `coordinators`, `syncmasters` & `syncworkers`
 because servers in these groups do not need persistent storage.
 
+### `spec.<group>.priorityClassName: string`
+
+Priority class name for pods of this group. Will be forwarded to the pod spec.
```

Comment:
Should we add a note that the class should be created upfront (potentially with link to k8s docu)?